templates/template.html

```diff
@@ -48,7 +48,10 @@
         </li>
         <li class="nav-item">
           <a class="nav-link" href="/donation">Donation</a>
-               </li>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/recent">recent articles</a>
```

Comment:
Thanks, @sanoussibarkay . 

`recent articles` should be `**R**ecent articles`.

Hui